dataservice/api/sequencing_center/resources.py

```diff
@@ -0,0 +1,157 @@
+from flask import abort, request
+from marshmallow import ValidationError
+
+from dataservice.extensions import db
+from dataservice.api.common.pagination import paginated, Pagination
+from dataservice.api.sequencing_center.models import SequencingCenter
+from dataservice.api.sequencing_center.schemas import (
+    SequencingCenterSchema
+)
+from dataservice.api.common.views import CRUDView
+
+
+class SequencingCenterListAPI(CRUDView):
+    """
+    SequencingCenter REST API
+    """
+    endpoint = 'sequencing_centers_list'
+    rule = '/sequencing-centers'
+    schemas = {'SequencingCenter': SequencingCenterSchema}
+
+    @paginated
+    def get(self, after, limit):
+        """
+        Get all sequencing_centers
+        ---
+        description: Get all sequencing_centers
+        template:
+          path:
+            get_list.yml
+          properties:
+            resource:
+              SequencingCenter
+        """
+        q = SequencingCenter.query
+
+        return (SequencingCenterSchema(many=True)
+                .jsonify(Pagination(q, after, limit)))
+
+    def post(self):
```

Comment:
We may consider an upsert here by checking to see if there is already a sequencing center with the same name, and returning that instead of creating a duplicate center.

Reply:
That would make sense if we go that route.